src/verify-and-receive.ts

```diff
@@ -1,14 +1,16 @@
+import AggregateError from "aggregate-error";
 import { verify } from "@octokit/webhooks-methods";
 
 import type {
+  EmitterWebhookEvent,
   EmitterWebhookEventWithStringPayloadAndSignature,
   State,
 } from "./types";
 
 export async function verifyAndReceive(
   state: State & { secret: string },
   event: EmitterWebhookEventWithStringPayloadAndSignature,
-): Promise<any> {
+): Promise<void> {
```

Comment:
Can you elaborate on why this becomes `void`?

Reply:
sure @oscard0m 

It is the return value of eventHandler.receive
see:
https://github.com/octokit/webhooks.js/blob/cc836d042288c5c1216089d3cc5f2afaa58dccb9/src/event-handler/index.ts#L29